src/components/ThanksForSuggesting.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <div v-show="NoResultsToShow">
```

Comment:
Moved the decision for whether to show this component to the containing component

Reply:
ah, thx!